src/helper.js

```diff
@@ -22,6 +25,29 @@ Helper.config = require(path.resolve(path.join(
 	"config.js"
 )));
 
+function getVersion() {
+	const gitCommit = getGitCommit();
+	return gitCommit ? `source (${gitCommit})` : `v${pkg.version}`;
+}
+
+let _gitCommit;
+function getGitCommit() {
+	if (_gitCommit !== undefined) {
+		return _gitCommit;
+	}
+	try {
+		_gitCommit = require("child_process")
+			.execSync("git rev-parse --short HEAD 2> /dev/null") // Returns hash of current commit
+			.toString()
+			.trim();
+		return _gitCommit;
+	} catch (e) {
+		// Not a git repository or git is not installed
+		_gitCommit = null;
+		return null;
+	}
+}
+
```

Comment:
Not too happy this was merged with removed comments as mentioned below some time ago, but oh well.